node/tessel-export.js

```diff
@@ -77,6 +77,15 @@ function Tessel(options) {
   this.version = 2;
 }
 
+Tessel.pwmPeriod = 0;
+Tessel.pwmPrescalars = [1, 2, 4, 8, 16, 64, 256, 1024];
+Tessel.pwmPrescalar = 0;
+// Actual lowest frequency is ~0.72Hz but 1Hz is easier to remember.
+// 5000 is the max because any higher and the resolution drops
+// below 7% (0xFFFF/5000 ~ 7.69) which is confusing
+Tessel.pwmMaxFrequency = 5000;
+Tessel.pwmMinFrequency = 1;
```

Comment:
Anywhere that's not user-code accessible. Considering the frequency and duty control safety requirements you outlined, there should be no way to touch these values directly.